src/utils.js

```diff
@@ -11,3 +11,19 @@ export function isEnterpriseLearner(user) {
 
   return false;
 }
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
nit: with 2 named exports in this file now, we can remove the eslint disabling! 😄